Sources/ArgumentParser/Parsable Properties/Option.swift

```diff
@@ -239,8 +239,16 @@ extension Option {
       let kind = ArgumentDefinition.Kind.name(key: key, specification: name)
       let help = ArgumentDefinition.Help(options: initial != nil ? .isOptional : [], help: help, key: key)
       var arg = ArgumentDefinition(kind: kind, help: help, parsingStrategy: ArgumentDefinition.ParsingStrategy(parsingStrategy), update: .unary({
-        (origin, _, valueString, parsedValues) in
-        parsedValues.set(try transform(valueString), forKey: key, inputOrigin: origin)
+        (origin, name, valueString, parsedValues) in
+        
+        do {
+            let transformedValue = try transform(valueString)
+            parsedValues.set(transformedValue, forKey: key, inputOrigin: origin)
+        }
+        catch /* All other errors */ {
```

Comment:
Nit: don't start a new line before `catch`, here and elsewhere.

```suggestion
        } catch {
```